src/labs/ViewLab.tsx

```diff
@@ -71,6 +71,21 @@ const ViewLab = () => {
     setNewNotes(notes)
   }
 
+  const deleteNote = async (noteIndexToDelete: number) => {
+    if (!labToView || !labToView.notes) {
+      return
+    }
+
+    const updatedNotes = labToView!.notes!.filter((_note, i) => i !== noteIndexToDelete)
```

Comment:
I think rather than removing it from the array, we should set a flag "deleted". 

This way the note remains for auditing purposes.

Reply:
I think add a deleted flag to the  `Notes` as our notes should be consistent regardless of where it being used; respective modules code will default it to false (not deleted) when the note is created. @jackcmeyer @tehKapa thoughts?